docs/documentation.md

```diff
@@ -24,7 +24,7 @@
 
   - [Prerequisites](prerequisites.md) - A more detailed view of the prerequisites needed to fully use the `control-repo`
 
-  - [Noop Mode](noop_mode.md) - An overview on how to enforce `noop mode server side` with this repo
+  - [Noop Mode](noop_mode.md) - An overview on how to enforce `server side` `noop mode` with this `control-repo`
```

Comment:
I've seen here and in the merged PR that you use single ticks ` to emphasize words.
As said eaerlier I prefer to use monospaced texts (with 1 or 3 ticks, not really sure about the difference if used inline) only to refer to computer strings (paths, commands, outputs) and eventually bold (1 asterisk) to emphasize words. In this case, the original format is already not "compliant" and in other occasions I'd have used an asterisk.
Given that your patches are already big improvements on the existing text, how would YOU emphasize words?

Reply:
Actually I meant 2 asterisks to emphasize in bold words which need emphasis